shuup/front/template_helpers/general.py

```diff
@@ -410,3 +411,14 @@ def can_toggle_all_seeing(context):
         # 'is all seeing' is purely person contact feature.
         return False
     return getattr(request.user, "is_superuser", False)
+
+
+@contextfunction
+def get_admin_url(context, object):
+    page = {}
```

Comment:
as we talked in slack, import this and use it https://github.com/shuup/shuup/blob/master/shuup/admin/template_helpers/shuup_admin.py#L136